src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md

```diff
@@ -57,26 +63,43 @@ $ k6 run -e MY_HOSTNAME=test.k6.io script.js
 
 </CodeGroup>
 
-Note: This can *not* be used to configure k6 with environment variables as listed on the [options](/using-k6/options) page. In other words `-e K6_ITERATIONS=120` will *not* configure the script [iterations](/using-k6/options#iterations), it will just provide `__ENV.K6_ITERATIONS` to the script, unlike `K6_ITERATIONS=120 k6 run script.js`.
+When setting both system environment variables and explicitly defining them, there is an order of [precedence](https://k6.io/docs/using-k6/options#using-options) that will overwrite options from the next level.
```

Comment:
I think this sentence should be dropped as it's confusing IMO in the middle of this section, especially with the `Note` below. 

It's basically saying that the section we are talking about actually doesn't have precedence, but if you miss reading the Note below, it kind of read like the reverse, and more like a reminder that you can configure k6 itself with env variables and that `-e` works for that, which it doesn't ;)

Reply:
I agree